### PR TITLE
Binabalami

### PR DESCRIPTION
**Pull request by binabalami**

### CODE REVIEW

File: bina.php [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?php
+echo "this is bina's code file";
```
Comment (sundarban1) [inline bina.php:2]: can you add full stop in the sentence?

**Comment by dummydeveloper000** [comment]
Well done!

**Comment by RabinChaudhary** [comment]
great job!

**Comment by surajdahal300** [comment]
nice work

**Pull request merged**